### DISCUSSION TITLE
Office hours

### DISCUSSION THREAD

**Discussion by romanlutz** [fairlearn/fairlearn#816] [General]
Since this was raised today I wanted to get the conversation started! What's a good way for us to "advertise" for office hours?

Some ideas:
- A GitHub discussion like this one. Problem: hardly discoverable...
- The website. Problem: updates would take a PR...
- Discord. Could be done easily without PR and on a dedicated channel. Also, Discord offers voice/video channels to do the office hours. We could point to Discord from GitHub and the website.
- Twitter. Can't hurt to mention this every so often.

@fairlearn/fairlearn-maintainers @kstohr

**Comment by kstohr** [comment]
... shared calendar? I can just drop the fairlearn calendar into mine and
see whatever events might be coming up... plan around them, etc. Other
options good too, but for me my calendar is my go to.

On Fri, May 21, 2021 at 8:04 AM Roman Lutz ***@***.***> wrote:

> Since this was raised today I wanted to get the conversation started!
> What's a good way for us to "advertise" for office hours?
>
> Some ideas:
>
>    - A GitHub discussion like this one. Problem: hardly discoverable...
>    - The website. Problem: updates would take a PR...
>    - Discord. Could be done easily without PR and on a dedicated channel.
>    Also, Discord offers voice/video channels to do the office hours. We could
>    point to Discord from GitHub and the website.
>    - Twitter. Can't hurt to mention this every so often.
>
> @fairlearn/fairlearn-maintainers
> <https://github.com/orgs/fairlearn/teams/fairlearn-maintainers> @kstohr
> <https://github.com/kstohr>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/fairlearn/fairlearn/discussions/816>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6WM546E6FKR7TWLZOCU63TOZZALANCNFSM45JKMXAA>
> .
>


-- 
Kas Stohr
+1 646 554 7671

**Reply by romanlutz** [reply]
So you mean Fairlearn should have a calendar to begin with 😄 
Are there solutions for this that don't require creating an email address? I wouldn't want yet another thing to check...
